lebedigital/openbis/README.md

```diff
@@ -0,0 +1,140 @@
+# Interbis/ExpStep
+
+## Motivation
+
+### Interbis/Expstep is an extension of the Pybis library meant for improving data synchronisation with an openBIS datastore.
+
+## Glossary
+
+- **Interbis** - A Subclass of the Openbis class in the pybis library. It is used for connecting to the openBIS Database and for all Database queries like getting an overview of the Database
+- **ExpStep** - A Class for working with openBIS Experimental Steps. Used for defining Objects which describe real Experiments that you want to describe with metadata and upload to the openBIS Database
+- **Space** - The Highest "directory" abstraction in the Datastore. Contains all other openBIS "directories". One Space is defined per User or per Group.
+  - _Example_: Some Paper about compressing concrete
+- **Project** - Second highest "directory" abstraction in the Datastore. Contained in a Space, you can define infinitely many of them.
+  - _Example_: Concrete_Compression
+- **Collection / Experiment** - Third highest "directory" abstraction in the Datastore. Contained in a Project, you can define infinitely many of them.
+  - _Example_: Concrete_Compression_Tests
+- **Sample / Object** - An abstract representation of your experiment with your defined metadata **also called Experimental Steps** in the openBIS Database. Contained in a Collection, you can define infinitely many of them.
+  - _Example_: Concrete_Compression_Experiment_17_10_2022
+- **Dataset** - An openBIS abstraction containing your data files like Excel sheets or csvs.
+  - _Example_: Concrete_Compression_Experiment_17_10_2022_RAW_DATA
+- **Sample / Object Type** - A blueprint you need to define in order to upload Samples/Objects of real world experiments.
+  - _Example_: EXPERIMENTAL_STEP_CONCRETE
+- **Property** - One piece of metadata you use do describe your Samples/Objects. 
+  - _Example_: Name of your sample
+- **Parents** of a Sample - You can define another Sample to be a parent of a different sample.
+  - _Example_: A concrete mixture could be a **parent** of a concrete compression experiment
```

Comment:
A conrete mixture object [Experimental_step_MIX] as parent of a concrete ?? experiment object [Experiment_step_??]

?? : siehe oben compression oder Emodul

Reply:
The Type EXPERIMENTAL_STEP_UCT is now used throughout the readme, same as the Sample / Object example